index.asciidoc

```diff
@@ -19,6 +19,7 @@ heterogeneous environments via a common framework and APIs.
 
 == Recent Updates
 
+- July-August, 2017: Updated firmware and software for the Intelligent Power Controller appliance, built from 42ITy(TM) source-code,  was released and made downloadable at link:http://www.eaton.eu/ipminfrastructure[Eaton IPM Infra version 1.2.0].
```

Comment:
July 31rst, 2017 is the date to consider for the official publication, as per my (internal) announcement mail

Reply:
Updated. We had a number of dates that could be tagged "it" :)